pkgs/applications/science/misc/root/cling.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, python, wget, fetchFromGitHub, libffi, cacert, git, cmake, llvm, ncurses, zlib, fetchgit, glibc, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "cling";
```

Comment:
Shouldn't the version be part of `name`?

Reply:
```suggestion
  pname = "cling";
```

Reply:
Also needs `version`

Reply:
Umm... I'm not sure that Cling has proper version numbers. The snapshots they provide at https://root.cern.ch/download/cling// just have names like `cling_2019-07-05_ubuntu18.tar.bz2`. I put `version = "2019-07-05";` unless anyone has a better idea

Reply:
This is documented in the manual: https://nixos.org/nixpkgs/manual/#sec-package-naming
The format for version will be `"unstable-YYYY-MM-DD" then.

Reply:
Oh I see, there's a `VERSION` file in the Cling repo with the current value `0.6~dev`. It hasn't been updated in 2 years though...

Reply:
So we can ignore it.

Reply:
My understanding that 2019-07-05 doesn't describe the version accurately. You build cling from `rev = "5789f5b2eb14ff8e14e6297fe3a449d0ef794f17"` which comes from 2018-11-20, there were many more commits in master branch since then.